makefile.shade

```diff
@@ -5,16 +5,6 @@ k-standard-goals
 
 default DOTNET_ARGUMENT='-v '
 
-#build-compile target='compile'
```

Comment:
And the `build-compile` target was actually replacing the default target defined in `_k-standard-goals.shade`, meaning the `experimental` projects were being built by default.